cms/management/commands/create_initial_data.py

```diff
@@ -48,15 +55,38 @@ def output(self, app_name, verbosity, *, done=False, result=False):
         if verbosity >= 2 and result:
             pprint.pprint(result)
 
+    def flush_handler(self, do_flush, verbosity):
+        if do_flush:
+            msg = ('You have provided the --flush parameter, this will cleanup '
+                'the database before creating new data.\n'
+                'Type \'y\' or \'yes\' to continue, \'n\' or \'no\' to cancel: '
+                )
+        else:
+            msg = (
+                'Note that this command won\'t cleanup the database before '
+                'creating new data.\n'
+                'If you would like to cleanup the database before creating '
+                'new data, \nplease type \'n\' or \'no\' and call the '
+                'create_initial_data command adding the --flush parameter\n'
+                'Type \'y\' or \'yes\' to continue, \'n\' or \'no\' to cancel: '
+            )
+        confirm = input(self.style.WARNING(msg))
+        if do_flush and (confirm in ('y', 'yes')):
```

Comment:
Style nit: (confirm in ('y', 'yes')) -> confirm in ('y', 'yes') (parentheses can be removed)